client/src/EventSubscriber/OrgCreatedSubscriber.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\EventSubscriber;
+
+use App\Event\OrgCreatedEvent;
+use App\Service\Audit\AuditEvents;
+use App\Service\Time\DateTimeProvider;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class OrgCreatedSubscriber implements EventSubscriberInterface
+{
+    /** @var DateTimeProvider */
+    private $dateTimeProvider;
+
+    /** @var LoggerInterface */
+    private $logger;
+
+    public function __construct(
+        LoggerInterface $logger,
+        DateTimeProvider $dateTimeProvider
```

Comment:
We can use constructor property promotion here to remove some boilerplate:

```suggestion
        private LoggerInterface $logger,
        private DateTimeProvider $dateTimeProvider
```

And then drop lines 15-19 and 25-26.